supporting_scripts/course-scripts/hyperion-benchmark-workflow/utils.py

```
@@ -0,0 +1,41 @@
+config.read(['config.ini'])
+
+CLIENT_URL: str = config.get('Settings', 'client_url')
+SERVER_URL: str = config.get('Settings', 'server_url')
+ADMIN_USER: str = config.get('Settings', 'admin_user')
+ADMIN_PASSWORD: str = config.get('Settings', 'admin_password')
+
+def login_as_admin(session: requests.Session) -> None:
+    """Authenticate as an admin using the provided session."""
+    authenticate_user(ADMIN_USER, ADMIN_PASSWORD, session)
+
+def authenticate_user(username: str, password: str, session: requests.Session = requests.Session()) -> requests.Response:
```

Ruff (0.14.6)

Do not perform function call `requests.Session` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)